tests/PHPStan/Type/TestDecimalOperatorTypeSpecifyingExtension.php

```diff
@@ -10,7 +10,7 @@ final class TestDecimalOperatorTypeSpecifyingExtension implements OperatorTypeSp
 
 	public function isOperatorSupported(string $operatorSigil, Type $leftSide, Type $rightSide): bool
 	{
-		return in_array($operatorSigil, ['-', '+', '*', '/'], true)
+		return in_array($operatorSigil, ['-', '+', '*', '/', '^'], true)
```

Comment:
`^` is XOR, POW is `**`.

Reply:
oh my, nice catch. it means, we have 2 uncovered cases :).

added POW for real now